layers/+config-files/nginx/packages.el

```diff
@@ -0,0 +1,18 @@
+;;; packages.el --- nginx layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Nathaniel Waisbrot <code@waisbrot.net>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst nginx-packages
+  '(
+    nginx-mode
+    ))
+
+(defun nginx/init-nginx-mode ()
+  (use-package nginx-mode))
```

Comment:
As a general recommendation, every package should be deferred, so adding `:defer t` would be better in my opinion. Unless it is strictly necessary to load it at startup time.

Let's hear other's opinions as well.